twenty-first/src/math/polynomial.rs

```diff
@@ -347,6 +353,28 @@ where
         Self::new(hadamard_product)
     }
 
+    pub fn batch_multiply(factors: &[Self]) -> Self {
+        factors.iter().fold(Self::one(), |l, r| l.multiply(r))
+    }
+
+    pub fn par_batch_multiply(factors: &[Self]) -> Self {
+        let num_threads = available_parallelism().map(|nzu| nzu.get()).unwrap_or(1);
```

Comment:
I find variable name `nzu` confusing. It requires knowing the underlying type to understand the name. If it were just `n.get()`, I could happily accept that the type of `n` is just anything from which I can `.get()` my value, without caring about the concrete type, and would not be confused.

Reply:
Fixed in 6a810ab8 by de-abbreviating to `non_zero_usize`.